src/client/components/Overview/Overview.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Overview.styles.css';
+
+// user List component not working and breaking all other functionalities
+// import UsersList from '../UsersListComponent/UsersList';
+import Search from '../Search/Search';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chat-title">Chats</h3>
+      <div className="search">
+        <Search />
+      </div>
+      <div className="users-list">{/* <UsersList /> */}</div>
```

Comment:
Dont outcomment this, we need to see the UserList in order to create a new conversation

Reply:
I need it ASAP!!! For my ticket https://github.com/HackYourFuture-CPH/chattie/issues/38

Reply:
Okay, we will merge. Then you can work on it victoria